src/printer.js

```diff
@@ -65,12 +64,14 @@ Printer.prototype.print = function(callback) {
 	async.eachSeries(
 		_self.commandQueue,
 		function(command, callback) {
-			if (_self.commandDelay !== 0) {
-				sleep.usleep(_self.commandDelay);
+			function write() {
+				_self.serialPort.write(command, function() {
+					_self.serialPort.drain(callback);
+				});
 			}
-			_self.serialPort.write(command, function() {
-				_self.serialPort.drain(callback);
-			});
+
+			if (_self.commandDelay === 0) return write()
```

Comment:
Hi,
Yes you're right, but for the consistency of code styling I would prefer the following

```javascript
if (_self.commandDelay === 0) {
  write();
}
else {
  setTimeout(write, Math.ceil(_self.commandDelay / 1000));
}
```

Thanks again for taking the time to investigate this issue.